crypto/bio/b_addr.c

```diff
@@ -760,8 +762,11 @@ int BIO_lookup_ex(const char *host, const char *service, int lookup_type,
                 he_fallback_address = INADDR_ANY;
                 break;
             default:
-                OPENSSL_assert(("We forgot to handle a lookup type!" == 0));
-                break;
+                /* We forgot to handle a lookup type! */
+                assert(0);
```

Comment:
Why not reuse original pattern, i.e. `assert("We forgot to handle a lookup type!" == NULL)`?